dhcpv4/server4/conn.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Maybe this should be renamed to `conn_unix.go`?

Reply:
done! thanks